### PR TITLE
SO-58

### PR DESCRIPTION
**Pull request by JDoan03**
SO-58
- create new projects functionality
- new project status enum
- New Project modal overlay
- Retrieving projects for members
- Minor changes to styles of other components
- Creation of new projects functionality
- new project status enum coinciding with database
- RLS updates on supabase as well, trigger function was DEFINER

### CODE REVIEW

File: src/app/(dashboard)/projects/page.tsx [hunk 1]
```diff
@@ -1,82 +1,99 @@
...
+	if (!org) {
+		redirect('/organization');
+	}
+	const roleResponse = await getOrganizationMemberRole(org);
+	const role: string = roleResponse?.role || '';
+
+	// if role supervisor or admin retrieve all projects
+	const projects = await getAllProjects(org, userInfo?.id);
+
+	// console.log(projects);
```
Comment (jvniorrr) [inline src/app/(dashboard)/projects/page.tsx]: Redundant code below, just return the page after retrieving projects. Have double page call in here.

File: src/components/projects/StatusBar/StatusBar.tsx [hunk 2]
```diff
@@ -10,7 +11,8 @@ const StatusBar = ({ status }: StatusBarProps) => {
 	return (
 		//generalized component circular
 		<button
-			className={`status-bar status-${status} rounded-full text-center px-4 py-2 font-bold`}
+			className={`status-bar status-${status}`}
+			// TODO: implement logic for filters; can pass fxn as prop
```
Comment (jvniorrr) [inline src/components/projects/StatusBar/StatusBar.tsx:15]: todo. new task

File: src/app/(dashboard)/projects/page.tsx (hunk 1 above)
Comment (JDoan03) [reply src/app/(dashboard)/projects/page.tsx]: Changes have removed the code

**Review by jvniorrr — APPROVED** [review]
AWESOME

**Pull request merged**